Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -70,11 +70,15 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
 
   // MARK: - Configuration
 
-  func configureWith(value: (amount: Double, currency: String)) {
+  func configureWith(value: PledgeDataSource.PledgeInputRow) {
+    guard case .pledgeAmount(let amount, let currencySymbol) = value else {
```

Comment:
This is temporary. The intention is to pipe this `value` directly into a view model for this cell:

```
  func configureWith(value: PledgeDataSource.PledgeInputRow) {
    self.viewModel.configure(with: value)
  }
```

And allow the cell's view model to match against the desired `PledgeInputRow` case.